frontend/src/components/search/components/common/filter-dropdown/filter-dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'common/types/types';
+import { useState } from 'hooks/hooks';
+import { IconName } from 'common/enums/components';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  filter: string;
+  onChangeFilter: (value: string) => void;
+  title: string;
+  options: {
+    text: string;
+    value: string;
+  }[];
+};
+
+const FilterDropdown: FC<Props> = ({ filter, onChangeFilter, title, options }) => {
```

Comment:
I think dropdown has to be closed on outside click or on scroll menu. Confirm with QA but I think it's needed

Reply:
@kyrylo-burmelov Had a talk with QA team. We decided to leave dropdowns open for outside click and during scroll.

Reply:
hrm... @VitaliiBedletskyi what do you think? I would recommend to close it at least on outside click

Reply:
@Barracuda713 @kyrylo-burmelov @HorkushaS We decided that needed!!!

Reply:
@kyrylo-burmelov @VitaliiBedletskyi @HorkushaS Ok, I'll add it)